PROTOCOL.md

```diff
@@ -2522,6 +2522,16 @@ be case-insensitive when processed by the server.
 This header can be used in the request for [Query Table Metadata](#query-table-metadata), 
 [Query Table](#read-data-from-a-table), and [Query Table Changes](#read-change-data-feed-from-a-table).
 
+### responseFormat
+Indicates the format to expect in the [API Response Format in Parquet](#api-response-format-in-parquet), two values are supported.
+
+- parquet: Represents the format of the delta sharing protocol that has been used in `delta-sharing-spark` 1.0 
+and less, also the default format if `responseFormat` is missing from the header. All the existing delta
+sharing connectors are able to process data in this format. 
+- **delta**: format can be used to read a shared delta table with minReaderVersion > 1, which contains 
+readerFeatures such as Deletion Vector or Column Mapping. `delta-sharing-spark` libraries 
+that are able to process `responseformat=delta` will be released soon.
+
```

Comment:
responseFormat
Specifies the expected format of the [API Response in Parquet](https://github.com/delta-io/delta-sharing/pull/734/files#api-response-format-in-parquet). Two values are supported:

parquet: Represents the response format used by delta-sharing-spark version 1.0 and earlier. This is the default format if responseFormat is not specified in the header. All existing Delta Sharing connectors are compatible with this format.

delta: Enables reading of shared Delta tables with minReaderVersion > 1, which may include advanced reader features such as Deletion Vectors or Column Mapping. Support for responseFormat=delta will be available in upcoming versions of the delta-sharing-spark library.

Reply:
updated.